### ISSUE TITLE
Help Needed – Unable to Fine-Tune Qwen3-VL

### ISSUE THREAD

**Issue by langge52** [modelscope/ms-swift#9324]
### Checklist / 检查清单

- [x] I have searched existing issues, and this is a new bug report. / 我已经搜索过现有的 issues，确认这是一个新的 bug report。

### Bug Description / Bug 描述

Hello,

I am trying to fine-tune the Qwen3-VL-8B-Thinking model using LoRA, but I keep running into an error during training. The error message indicates that the label exceeds the number of classes.

Training Command:

### How to Reproduce / 如何复现

Training Command:
CUDA_VISIBLE_DEVICES=2,3 swift sft \
--model Qwen/Qwen3-VL-8B-Thinking \
--tuner_type lora \
--dataset train/jsonl/path \
--val_dataset test/jsonl/path \
--torch_dtype bfloat16 \
--num_train_epochs 10 \
--per_device_train_batch_size 2 \
--per_device_eval_batch_size 2 \
--learning_rate 1e-4 \
--lora_rank 16 \
--lora_alpha 32 \
--target_modules all-linear \
--gradient_accumulation_steps 16 \
--eval_steps 50 \
--save_steps 50 \
--save_total_limit 3 \
--logging_steps 5 \
--max_length 8192 \
--warmup_ratio 0.05 \
--dataloader_num_workers 4 \
--model_author swift \
--model_name swift-robot \
--output_dir output/qwen3_vl_med_diagnosis \
--model_kwargs '{"visual": {"max_dynamic_patch": 1}}' > train1.log 2>&1 &

### Additional Information / 补充信息

Example JSONL Sample:
{
  "messages": [
    {"role": "user", "content": "<image> Assess the development of the organ across all slices and provide your judgment."},
    {"role": "assistant", "content": "xxxxxx"}
  ],
  "images": ["image path"],
  "label": 1
}

**Comment by hjh0119** [comment]
remove label column
